roles/datadog/tasks/pkg-debian.yml

```diff
@@ -52,14 +51,12 @@
     name: "datadog-agent={{ datadog_agent_version }}"
     state: present
     force: "{{ datadog_agent_allow_downgrade }}"
-    update_cache: true
```

Comment:
Partially addressed by d3841652113ae51a1c7469d2cb313b50121de43e. Thinking it through, with the logic here it might be more useful to get the failure on the task that's trying to add the new repo to the list (on any given run, only one of the two tasks will execute). Still thinking about this one . . . Also wondering how the datadog role passes idempotency tests . . . I'll try fixing the rest of the CI errors, then come back to this.